src/components/search/SearchResults.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {useDispatch, useSelector} from 'react-redux';
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import {Link} from 'react-router-dom';
 import {setGetReferenceCurieFlag, setReferenceCurie} from '../../actions/biblioActions';
 import {Modal} from 'react-bootstrap';
-import {setSearchError} from '../../actions/searchActions';
+import {setSearchError, searchXref} from '../../actions/searchActions';
 import Button from 'react-bootstrap/Button';
 
+const XrefElement = (xref) => {
+    const [url, setUrl] = useState(null);
+    useEffect(() => {
+      searchXref(xref.xref.curie, setUrl);
+    }, [url]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
searchXref modifies url and we want to execute the useEffect only once, so url should be removed from the useEffect deps